helm/csi-vxflexos/templates/node.yaml

```diff
@@ -203,6 +203,10 @@ spec:
             - name: X_CSI_HEALTH_MONITOR_ENABLED
               value: "{{ .Values.node.healthMonitor.enabled }}"
             {{- end }}
+            {{- if eq .Values.node.approveSDCEnabled true}}
```

Comment:
should we check if this key exists first?
if it doesn't - set value to false or don't set at all

Reply:
done